### ISSUE TITLE
Why `Outline`'s props `patternTexture` type is number?

### ISSUE THREAD

**Issue by Suprhimp** [pmndrs/react-postprocessing#182]
Hi, I'm very happy to use r3f and react-postprocessing now with my project.

By the way, I found something strange about the `OutlineEffect`.

I tested the props `patternTexture` in OutlineEffect, I found that `patternTexture `type is `number`. When I follow the `OutlineEffect` from `postprocessing`, patternTexture is not number. I think it should be `THREE.Texture`.

When I just put  my pattern image to `patternTexture` props, it just cover my whole scene like below.
![image](https://user-images.githubusercontent.com/73486185/224232318-203c593e-5294-403d-a7ec-181e24df7fa6.png)

+ I think we should allow `visibleEdgeColor`,`hiddenEdgeColor` props type `string`.

**Comment by talentlessguy** [comment]
Most likely old version of pp has this prop as a number

we're going to release #179 and #180 soon with latest pp and it should eliminate some problems

**Comment by Suprhimp** [comment]
@talentlessguy  still, the patternTexture type is number..

**Comment by talentlessguy** [comment]
@Suprhimp is it still a number? rpp now uses strictly pp types

**Comment by Suprhimp** [comment]
Yep I guess.. I want to fix and make a PR with that but I don't know what should I fix first... 

https://github.com/pmndrs/postprocessing/blob/3e6ea9d8fddd2ca2d269e2182752f8199b8248af/src/effects/OutlineEffect.js#L24
